src/pages/Login/Login.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import Button from '../../components/Button/Button';
+import Input from '../../components/Input/Input';
+
+const Login = () => {
+  const [userInfo, setUserInfo] = useState({
+    email: '',
+    password: '',
+  });
+  const navigate = useNavigate();
+
+  const handleUserInfo = e => {
+    const { name, value } = e.target;
+    setUserInfo({ ...userInfo, [name]: value });
+  };
+
+  const emailRegex =
+    /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i;
+  const pwRegex = /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,15}$/;
+
+  const isEmailValid = emailRegex.test(userInfo.email);
+  const isPwValid = pwRegex.test(userInfo.password);
+  const isValidCheck = isEmailValid && isPwValid;
+
+  const handleLogin = e => {
+    e.preventDefault();
+
+    fetch('http://10.58.52.243:8000/users/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email: userInfo.email,
+        password: userInfo.password,
+      }),
+    })
+      .then(response => {
+        if (response.status === 200) {
+          return response.json();
+        } else if (response.status === 400) {
+          alert('아이디, 비밀번호를 확인해주세요');
+        } else {
+          alert('로그인 실패');
+        }
+      })
+      .then(result => {
+        if (result.token) {
+          localStorage.setItem('token', result.token);
+        }
+        if (result.nickname) {
+          localStorage.setItem('nickname', result.nickname);
+        }
+        if (result.message === 'LOGIN_SUCCESS') {
+          navigate('/');
+        }

```

Comment:
status code가 200인 경우에 result에 token, nickname이 있고, message가 'LOGIN_SUCCESS'인 것이 명확하다면 굳이 조건부로 처리하지 않으셔도 됩니다.
```suggestion
        localStorage.setItem('token', result.token);
        localStorage.setItem('nickname', result.nickname);
        navigate('/');
```

Reply:
수정했습니다